which-key.el

```diff
@@ -2030,6 +2049,48 @@ Finally, show the buffer."
                    (which-key--create-pages formatted-keys))
              (which-key--show-page 0)))))
 
+(defun which-key--translate-god-mode-key-vector (key-vector)
```

Comment:
This function looks similar to https://github.com/chrisdone/god-mode/blob/master/god-mode.el#L201. Do you have any idea that can get rid of the `read-event` call [on this line](https://github.com/chrisdone/god-mode/blob/master/god-mode.el#L225)? If so, we can just use the function `key-string-after-consuming-key` from `god-mode.el` instead of defining this one.